### DISCUSSION TITLE
Add sbctl to Debian repositories

### DISCUSSION THREAD

**Discussion by hueychen27** [Foxboron/sbctl#351] [Ideas]
Is there any motivation to publish `sbctl` to Debian and possibly other distros? It would be nice to only need to use our local package managers instead of using git` directly.

**Comment by Foxboron** [comment]
https://salsa.debian.org/go-team/packages/sbctl

It needs an maintainer willing to deal with it and upload it.

**Reply by Foxboron** [reply]
Also see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1030845

**Reply by hueychen27** [reply]
One day that person will come...
